src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/CallbackInvoker.java

```diff
@@ -172,22 +171,21 @@ private AboutToSubmitCallbackResponse validateAndSetFromAboutToSubmitCallback(fi
 
         validateSignificantItem(aboutToSubmitCallbackResponse, callbackResponse);
         callbackService.validateCallbackErrorsAndWarnings(callbackResponse, ignoreWarning);
+        callbackResponse.updateCallbackStateBasedOnPriority();
```

Comment:
add comment saying this is temporary and for backward compatibility and will be removed as part of the new ticket